pyproject.toml

```diff
@@ -16,10 +16,6 @@ Home = "https://github.com/allisonkarlitskaya/systemd_ctypes/"
 [tool.mypy]
 mypy_path = 'src'
 
-[tool.pytest.ini_options]
-pythonpath = ["src"]
-log_cli = true
```

Comment:
This breaks running `pytest`. The commit message says so, but this feels unnecessary and evil. This makes debugging failures annoying, as it's yet another layer to peel off (aside from having to install yet another big thing into the test environment).

If you really insist on this, please update README, as it's no longer true.

Reply:
I brought this back.